pennylane_ionq/device.py

```diff
@@ -286,12 +336,14 @@ def __init__(
         self,
         wires,
         *,
-        target="qpu",
-        backend=None,
         gateset="qis",
         shots=1024,
+        backend="harmony",

```

Comment:
changed the default `backend` to "harmony" because every job submission was raising a warning unless the user specifies "harmony" (or some other backend). Since I believe this was effectively the default before, this will keep that behaviour and not give users warnings (unless they specify `None`, in which case the warning is justified)